### PR TITLE
chore: Modify update-apk script

### LINKED ISSUE #1328: Modify the update-apk script
### Chore

**Description**

According to the points discussed in the meeting the following things need to be changed in the update-apk script

- For the apk's (dev ones) that get generated after merging a PR their names need to be changed to a more informative names For eg.eventyay-organizer-dev-app-debug apk.

- The master apk's should only be generated when the development is merged with master. 
(I think that this is already happening but the whenever a PR gets merged the commit message shows that master apk has also been created )

**Would you like to work on the issue?**
Yes

### PR DESCRIPTION
**Pull request by sridharjajoo**
Fixes #1328 

Checklist:

- [x] I have checked for PMD and check-style issues 
- [x] I have read the [Contribution & Best practices Guide](https://blog.fossasia.org/open-source-developer-guide-and-best-practices-at-fossasia) and my PR follows them.
- [x] My branch is up-to-date with the Upstream development branch.

### CODE REVIEW

**Comment by sridharjajoo** [comment]
Only the dev-apk name had to be changed

**Comment by iamareebjamal** [comment]
This is removing the files. Where is the changing code?

**Comment by sridharjajoo** [comment]
I missed that. I have done the changes

File: scripts/update-apk.sh [hunk 1]
```diff
@@ -18,7 +18,7 @@ cd apk
 if [ "$TRAVIS_BRANCH" == "$PUBLISH_BRANCH" ]; then
 	/bin/rm -f *
 else
-	/bin/rm -f app-playStore-debug.apk app-playStore-release.apk app-fdroid-debug.apk app-fdroid-release.apk
+	/bin/rm -f eventyay-organizer-dev-app-playStore-debug.apk eventyay-organizer-dev-app-playStore-release.apk eventyay-organizer-dev-app-fdroid-debug.apk eventyay-organizer-dev-app-fdroid-release.apk
```
Comment (iamareebjamal) [inline scripts/update-apk.sh]: `/bin/rm -f eventyay-organizer-dev-*.apk`

File: scripts/update-apk.sh [hunk 2]
```diff
@@ -43,6 +43,12 @@ if [ "$TRAVIS_BRANCH" == "$PUBLISH_BRANCH" ]; then
     done
 fi
 
+if [ "$TRAVIS_BRANCH" == "$DEPLOY_BRANCH" ]; then
+    for file in app*; do
+          cp $file eventyay-organizer-dev-${file%%}
```
Comment (iamareebjamal) [inline scripts/update-apk.sh]: mv

**Comment by sridharjajoo** [comment]
@iamareebjamal Done the changes

**Comment by iamareebjamal** [comment]
The goal of GSoC is to get permanent addition to teams in Open Source projects. It's sad to see everyone leave after the party when there is time when we need people. Although GSoC is over and it is completely up to you guys if you want to continue contributing to the project, but I believe that it would be beneficial for both you and the project if you continue to add support to it at your own pace.

This PR is open for days. Anyone up to finalize it @Masquerade0097 @sridharjajoo @mishuvs

**Comment by sridharjajoo** [comment]
@iamareebjamal I will try to finalize it today. Can you tell me as to why would it fail so that I can make the required changes?

**Comment by iamareebjamal** [comment]
The last line does not have the changed names

**Comment by sridharjajoo** [comment]
@iamareebjamal I have done some changes. Kindly review

**Pull request merged**